README.md

```diff
@@ -14,13 +14,17 @@ SPDX-License-Identifier: MIT OR Apache-2.0
 ![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/kdab/cxx-qt/github-cxx-qt-tests.yml)
 ![License (MIT/Apache2.0)](https://img.shields.io/crates/l/cxx-qt)
 [![REUSE status](https://api.reuse.software/badge/github.com/KDAB/cxx-qt)](https://api.reuse.software/info/github.com/KDAB/cxx-qt)
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](CODE_OF_CONDUCT.md)
+
 | Crate | Status |
 | --- | --- |
 | cxx-qt | [![docs.rs](https://img.shields.io/docsrs/cxx-qt?logo=docsdotrs)](https://docs.rs/cxx-qt) [![Crates.io](https://img.shields.io/crates/v/cxx-qt)](https://crates.io/crates/cxx-qt) |
 | cxx-qt-build | [![docs.rs](https://img.shields.io/docsrs/cxx-qt-build?logo=docsdotrs)](https://docs.rs/cxx-qt-build) [![Crates.io](https://img.shields.io/crates/v/cxx-qt-build)](https://crates.io/crates/cxx-qt-build) |
 | cxx-qt-lib | [![docs.rs](https://img.shields.io/docsrs/cxx-qt-lib?logo=docsdotrs)](https://docs.rs/cxx-qt-lib) [![Crates.io](https://img.shields.io/crates/v/cxx-qt-lib)](https://crates.io/crates/cxx-qt-lib) |
 | qt-build-utils | [![docs.rs](https://img.shields.io/docsrs/qt-build-utils?logo=docsdotrs)](https://docs.rs/qt-build-utils) [![Crates.io](https://img.shields.io/crates/v/qt-build-utils)](https://crates.io/crates/qt-build-utils) |
 
+There is a Zulip chat available at [https://cxx-qt.zulipchat.com](https://cxx-qt.zulipchat.com).
```

Comment:
```suggestion
If you want to get in touch with us, feel free to join our Zulip Chat at:
[https://cxx-qt.zulipchat.com](https://cxx-qt.zulipchat.com).
There we openly discuss all things about CXX-Qt development.
```

I think that sounds a bit more welcoming :)

Reply:
done :-)